src/app/core-ui/main-header/main-header.component.html

```diff
@@ -248,6 +248,22 @@
       }
     }
 
+    <button
```

Comment:
UI wise it might be cool to have some indication if there are filters or grouping active. But that is maybe something for a followup on this.